thunder/tests/test_grad.py

```diff
@@ -1944,35 +1944,6 @@ def func(x):
     torch.testing.assert_close(actual_gr, expected_gr)
 
 
-@pytest.mark.parametrize("device", ("cuda", "cpu"))
-def test_backward_recomputation_decomposed_ops(device):
```

Comment:
This test is for an old experimental feature `auto_recompute_intermediates` that has not caught on, so I'm removing the test here.